doc/GuideForCommitters.md

```diff
@@ -21,13 +21,41 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 
 # Eclipse OMR Committer Guide
 
-This document outlines some general rules for Eclipse OMR committers to
-follow when reviewing and merging pull requests and issues.  It also provides
-a checklist of items that committers must ensure have been completed prior to
-merging a pull request.
+This document provides information useful for Eclipse OMR committers.
 
+It outlines minimum participation requirements for committers, general rules
+for committers to follow when reviewing and merging pull requests and issues,
+and provides a checklist of items that committers must ensure are completed
+prior to merging a pull request.
 
-## General Guidelines
+## Minimum Committer Participation Standard
```

Comment:
Does this has to be the first section? It doesn't feel very welcoming to start off with "here are the criteria to avoid being kicked out". Committers should be familiar with this entire document, so maybe we could tack it on at the end as hopefully something most active committers won't really need to look at often?

Reply:
Fixed.